src/lib/Auth.ts

```diff
@@ -50,13 +52,32 @@ export const getApp = async () => {
     return { app };
 };
 
+function startTimer(){
+    Debug("Server timer starts")
+    Auth.timer_id = setTimeout(() => {
+        console.log(chalk.magenta('Server timeout: Please run fdk login command again.'));
```

Comment:
Don't use console.log here @manishprajapati52

Reply:
@vivek-gofynd 
This is debug log. I have added for debug purpose(debug.log file). Should I remove this?

Reply:
Added in Debug log too...